src/runtime_src/core/edge/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ add_subdirectory(common)
 add_subdirectory(user)
```

Comment:
Copyright date needs to be updated (line 2).

Reply:
Thank you. Updated the copyright date